### PR TITLE
fix local installation workaround

### PR DESCRIPTION
**Pull request by ggaaooppeenngg**
Signed-off-by: Peng Gao <peng.gao.dut@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/426)

### CODE REVIEW

**Comment by IronPan** [comment]
Thanks Peng. Note that pipeline is on its track to deprecate the bootstrapper in favor of the kubeflow deployment. 
https://www.kubeflow.org/docs/started/getting-started-gke/#deploy-kubeflow-on-gke-using-the-command-line

This will be available in https://github.com/kubeflow/kubeflow/releases/tag/v0.3.4-rc.1 Feel free to give a try

**Comment by ggaaooppeenngg** [comment]
I am actually manually deploying pipelines with deploy.sh, good to know that.

File: pipeline/pipeline/parts.yaml [hunk 1]
```diff
@@ -1,17 +1,3 @@
-// Copyright 2018 Google LLC
```
Comment (Ark-kun) [inline pipeline/pipeline/parts.yaml]: Needs to be replaces with hash signs.

**Comment by vicaire** [comment]
Is this PR still needed? Looks like there are some conflicts and the last update was quite a while ago.

**Pull request closed without merging**